src/api/auth/index.ts

```diff
@@ -0,0 +1,25 @@
+import { StorageKeys } from "../../constants";
```

Comment:
The api folder structure should match the exact path of the API. For example `/api/user DELETE` belongs in `/api/user/delete.ts`. 

In this case this logic should exist in `/api/signout.ts`